docs/Usage.md

```diff
@@ -89,9 +89,6 @@ export default initializeServer(createApp, {
 
     // Wire up logic for route-level vuex modules?
     vuexModules: true,
-    // Provide a function which will return a promise of all initial
-    // i18n translations to be included
-    i18nLoader: null,
```

Comment:
This was from _waaaay_ early on I think before we made initial translations entirely handled by the app.  I think in the beginning we thought we would need to do hat from within `vue-ssr-build` but it can be done entirely in `entry-client.js` and `createApp`